aggregation/pom.xml

```diff
@@ -0,0 +1,41 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+	http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>prof</artifactId>
+    <groupId>fk</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <groupId>fk.prof</groupId>
+  <artifactId>aggregation</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>Common impl for fk-prof</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>20.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>3.1.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.koloboke</groupId>
```

Comment:
This is used in MethodIdLookup, ProfileWorkInfo, RecordedProfileIndexes, basically wherever a concurrent map impl is not required